### PR TITLE
Feature/map locations

### PR DESCRIPTION
**Pull request by barakeitan**

### CODE REVIEW

File: controllers/pages.controller.js [hunk 1]
```diff
@@ -29,6 +31,21 @@ exports.renderContactUs = (req, res) => {
...
 }
 
+exports.HandleMapLocations = async (req, res) => {
+    const successMsg = req.flash("success")[0];
+    const errorMsg = req.flash("error")[0];
+
+    try
+    {
+        const locations = await Location.find({});
+        console.log(locations);
```
Comment (eladaviv) [inline controllers/pages.controller.js:41]: delete log

File: index.js [hunk 2]
```diff
@@ -61,7 +61,7 @@ app.use(async (req, res, next) => {
     res.locals.isAdmin = res.locals.login && req.user?.userType === 'Admin';
     res.locals.session = req.session;
     res.locals.currentUser = req.user;
-    const categories = await Category.find({}).sort({ title: 1 }).exec();
+    const categories = await Category.find({}).sort({ _id: 1 }).exec();
```
Comment (eladaviv) [inline index.js:64]: whats the difference?

File: public/js/map.js [hunk 3]
```diff
@@ -1,11 +1,22 @@
...
-      map: map
+  var locs=[];
+    $.get('/pages/map-location', function(data, status) {
+      console.log("data is:", data);
+      console.log("data.locations is : ", data.locations)
+      for (let index = 0; index < data.locations.length; index++) {
+        const element = data.locations[index];
+        locs.push({lat:element.lat, lng:element.lng});
+      }
+      console.log("locs[0] : ", locs[0])
```
Comment (eladaviv) [inline public/js/map.js:10]: delete logs

**Review by eladaviv — CHANGES_REQUESTED** [review]

File: index.js (hunk 2 above)
Comment (barakeitan) [reply index.js:64]: the nav bar items changed from home page to other pages. this fixes it

**Review by eladaviv — APPROVED** [review]

**Review by shirbalas — APPROVED** [review]

**Pull request merged**